src/langchain_google_bigtable/document_loader.py

```diff
@@ -110,6 +121,9 @@ def __init__(
         self.content_column_family = content_column_family
         self.content_column_name = content_column_name
         self.metadata_mappings = metadata_mappings
+        self.metadata_as_json_column_family = metadata_as_json_column_family
+        self.metadata_as_json_column_name = metadata_as_json_column_name
+        self.metadata_as_json_encoding = metadata_as_json_encoding
```

Comment:
Should we throw an error if one of these is specified but not the other? 

And probably verify they exist?

Reply:
We don't verify any of the mappings exist. If we find them - we populate them.
I could check, similarly to how I do it in the ChatMessageHistory. Should I?

Reply:
Done